### PR TITLE
Use BS from GT for 2022, 2023 and 2024 HI wfs

### PR DESCRIPTION
**Pull request by AdrianoDee**
#### PR description:

From the combination of:

- https://github.com/cms-sw/cmssw/pull/45346
- https://github.com/cms-sw/cmssw/pull/43242
- https://github.com/cms-sw/cmssw/pull/43197

we should be able to load the BS from the GT also for HI workflows. This PR proposes to do that.

@cms-sw/alca-l2  @francescobrivio @mandrenguyen @stahlleiton

### CODE REVIEW

**Comment by AdrianoDee** [comment]
please test

**Comment by AdrianoDee** [comment]
test parameters:
- workflows=159,160,161

**Comment by AdrianoDee** [comment]
please test

**Comment by AdrianoDee** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -36,11 +36,11 @@
 step1Up2018HiProdDefaults = merge ([{'--eventcontent':'RAWSIM'},step1Up2018HiDefaults])
 # step1 gensim: for 2022 HI prod
 step1Up2022HiProdDefaults = merge ([{'--conditions':'auto:phase1_2022_realistic_hi','--era':'Run3_pp_on_PbPb','--beamspot':'Realistic2022PbPbCollision','--eventcontent':'RAWSIM','--geometry':'DB:Extended'},step1Up2018HiDefaults])
-step1Up2023HiProdDefaults = merge ([{'--conditions':'auto:phase1_2023_realistic_hi','--era':'Run3_pp_on_PbPb_2023','--beamspot':'Realistic2023PbPbCollision','--eventcontent':'RAWSIM','--geometry':'DB:Extended'},step1Up2018HiDefaults])
+step1Up2023HiProdDefaults = merge ([{'--conditions':'auto:phase1_2023_realistic_hi','--era':'Run3_pp_on_PbPb_2023','--beamspot':'DBrealistic','--eventcontent':'RAWSIM','--geometry':'DB:Extended'},step1Up2018HiDefaults])
```
Comment (mmusich) [inline Configuration/PyReleaseValidation/python/relval_steps.py:39]: Judging from the [test results](https://github.com/cms-sw/cmssw/pull/45695#issuecomment-2289726275), `auto:phase1_2023_realistic_hi` (a.k.a. 

https://github.com/cms-sw/cmssw/blob/1e818e702bdf5b4bd93a80e5a8edceb6638d7146/Configuration/AlCa/python/autoCond.py#L91
) has a wrong reco beamspot vs sim beamspot pairing:

  * `SimBeamSpot_Realistic2022PbPbCollisionVtxSmearingParameters_v2_mc` (https://cern.ch/m8gf4)

image

* `BeamSpotObjects_Realistic2023PbPbCollisions_v2_mc` (https://cern.ch/crj4k)

image

@cms-sw/alca-l2 FYI

**Comment by perrotta** [comment]
Thank you @mmusich for noticing it.
Indeed, `phase1_2023_realistic_hi` was not updated in autoCond in previous #45346, which only updated the 2024 HI GT. #45704 now fixes it.

**Comment by AdrianoDee** [comment]
please test
(#45704 has been included in the latest IB)

**Comment by AdrianoDee** [comment]
Thanks to #45704 we went from

image

to 

image

so I would say this is ready to go.

**Comment by AdrianoDee** [comment]
+pdmv
2022, 2023, 2024 HI wfs show no differences

**Comment by srimanob** [comment]
+Upgrade

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**